client/src/app/site/pages/meetings/pages/assignments/modules/assignment-poll/components/assignment-poll-meta-info/assignment-poll-meta-info.component.html

```diff
@@ -28,14 +28,14 @@
                         {{ 'Candidates' | translate }}):
                         @if (enumerateCandidates) {
                             <ol>
-                                @for (title of poll.options[0]?.contentTitlesAsSortedArray; track title) {
+                                @for (title of poll.options[0]?.contentTitlesAsSortedArray; track title.title) {
```

Comment:
AFAIK title is not guaranteed to be unique in that case. In doubt you can always use `track $index`

Reply:
use now $index